carsus/io/opacities/opacities.py

```diff
@@ -0,0 +1,170 @@
+import logging
+import pandas as pd
+
+from carsus.io.util import read_from_buffer
+
+
+h_minus_URL = "https://raw.githubusercontent.com/tardis-sn/carsus-data-negative-ions/main/h_minus_cross_section_wbr.dat"
+
+logger = logging.getLogger(__name__)
+
+
+class HMINUSOPACITIESReader(object):
+    """
+    Class for extracting wavelengths and cross-section from h-minus opacities data files
+
+    Attributes
+    ----------
+    fname: path to h_minus_cross_section_wbr.dat
+
+    Methods
+    --------
+    h_minus_raw:
+        Return pandas DataFrame representation of h_minus_cross_section_wbr
+
+    """
+
+    h_minus_columns = ["wavelength", "cross_section"]
+
+    def __init__(self, fname=None):
+        """
+        Parameters
+        ----------
+        fname: str
+            Path to the h_minus file (http or local file).
+
+        priority: int, optional
```

Comment:
No priority argument passed in the declaration. Is this necessary for carsus? Either this should be added to the declaration or removed from the documentation.

Reply:
I don't think it is needed. I would just remove it from the docstring.